### ISSUE TITLE
Performance decrease after 6.9 due to JSON.parse of already json string

### ISSUE THREAD

**Issue by dzirtusss** [shakacode/react_on_rails#819]
There is some performance decrease for RoR versions after 6.9 which looks like this (from our internal server-side benchmarks):
```
----------------
 Benchmarking master VS test-ror
----------------
visitor_test1              79.751ms VS 92.823ms, +16.39% BAD !!!
visitor_test2              433.685ms VS 493.804ms, +13.86% BAD !!!
visitor_test3              123.157ms VS 134.358ms, +9.09% BAD !!!
user_test1                 131.78ms VS 144.51ms, +9.66% BAD !!!
user_test2                 1038.557ms VS 1128.468ms, +8.65% BAD !!!
user_test3                 162.683ms VS 174.69ms, +7.38% BAD !!!
```
The reason for it is JSON.parsing of the already json string to hash (and later back), which should be generally just a copy-paste operation
```
props.is_a?(String) ? JSON.parse(ERB::Util.json_escape(props)) : props
```
https://github.com/shakacode/react_on_rails/blob/master/lib/react_on_rails/react_component/options.rb#L17

**Comment by justin808** [comment]
Reverting the line here:

https://github.com/shakacode/react_on_rails/commit/7ccd603683af377880d4e16f56752ac7587c38b7#diff-ab00ff0a0648717a06431522f7eb6861R16

**Comment by justin808** [comment]
Fixed by #821 and shipped in 7.0.0.
